unity/Holo/ProjectSettings/ProjectSettings.asset

```diff
@@ -540,7 +541,7 @@ PlayerSettings:
   scriptingDefineSymbols: {}
   platformArchitecture: {}
   scriptingBackend:
-    Metro: 2
+    Metro: 1
   il2cppCompilerConfiguration: {}
```

Comment:
Changes Metro from 2 to 1. I'll have to run Unity to see what it actually means in UI :)